### ISSUE TITLE
Ubuntu 22.04 support?

### ISSUE THREAD

**Issue by heinowalther** [45Drives/cockpit-ceph-deploy#12]
Hi there
I am trying this plugin on Ubuntu 22.04, yet it fails when running the first step of the ansible script where it runs the /tmp/setup on the storage nodes...
I have tried to run this script by hand, where it asks to confirm that you want to add the focal repos...
But even after that has been run on all storage nodes, it still fails at the same step because the script asks then even if the repos is already present...
"You are on an unsupported version of Debian. Would you like to use 'focal' packages? [y/N]"

Anyway, I would really like to use this on 22.04 and not 20.04 for obvious reasons..  is there a version in the works?

**Comment by heinowalther** [comment]
A little update...  I have now also tried with Rocky 9.1 which also has problems... 
I tried to enable the test repos, and was able to "force" install with --nobest...
But it seems like the ansible_runner script is missing from the path... as it fails the checks... 

So basically you are more of less "forced" to use older linux variants to be able to use this deploy tool which I think is a bit sad, as it looks great...   I hope it will be updated soon...?

**Comment by LennyKopisch** [comment]
Hey
I have also tried Linux Rocky, the new 9 version has not worked for me either. However, the old one works, the 8 server. But still has support until 2029. is certainly not the optimal solution but if you are still looking for one.

**Comment by aniel300** [comment]
any update on this? any plans to support ubuntu 22, debian 12 or Rocky 9?

**Comment by LennyKopisch** [comment]
> any update on this? any plans to support ubuntu 22, debian 12 or Rocky 9?

I actually don't think so. Would not recommend it anymore. sadly...

**Comment by aniel300** [comment]
> > any update on this? any plans to support ubuntu 22, debian 12 or Rocky 9?
> 
> I actually don't think so. Would not recommend it anymore. sadly...

do u have any alternative to this?

**Comment by LennyKopisch** [comment]
Not really, used or wanted to use it for ceph installation, was more relaxed but went that way too. I can advise you, if you only need it for that just do it that way, here you get meanwhile ansible errors and several other problems. You don't want to deal with that.

**Comment by LennyKopisch** [comment]
> > > any update on this? any plans to support ubuntu 22, debian 12 or Rocky 9?
> > 
> > 
> > I actually don't think so. Would not recommend it anymore. sadly...
> 
> do u have any alternative to this?

Not really, used or wanted to use it for ceph installation, was more relaxed but went that way too. I can advise you, if you only need it for that just do it that way, here you get meanwhile ansible errors and several other problems. You don't want to deal with that.

**Comment by aniel300** [comment]
have u tried setting it up using proxmox?

**Comment by LennyKopisch** [comment]
> have u tried setting it up using proxmox?

I have run once on a VM for, proxmox and also once externally via a datastore for proxmox which I wanted to connect via iscsi. Conclusion: did not really work with this and I was mainly busy with config errors (not on my part).

**Comment by aniel300** [comment]
did u take a look at this? https://forum.proxmox.com/threads/ceph-single-node-vs-zfs.73924/post-385542

**Comment by LennyKopisch** [comment]
> did u take a look at this? https://forum.proxmox.com/threads/ceph-single-node-vs-zfs.73924/post-385542

yea I know this discussion, you can run ceph on a single node but it is useless. if that is the case use ZFS.

**Comment by aniel300** [comment]
> > did u take a look at this? https://forum.proxmox.com/threads/ceph-single-node-vs-zfs.73924/post-385542
> 
> yea I know this discussion, you can run ceph on a single node but it is useless. if that is the case use ZFS.

cant u start like this and then expand to multi nodes redundance?

**Comment by LennyKopisch** [comment]
> > > did u take a look at this? https://forum.proxmox.com/threads/ceph-single-node-vs-zfs.73924/post-385542
> > 
> > 
> > yea I know this discussion, you can run ceph on a single node but it is useless. if that is the case use ZFS.
> 
> cant u start like this and then expand to multi nodes redundance?

sure
